yt_dlp/extractor/common.py

```diff
@@ -731,6 +731,15 @@ def _initialize_geo_bypass(self, geo_bypass_context):
                 self._downloader.write_debug(
                     f'Using fake IP {self._x_forwarded_for_ip} ({country.upper()}) as X-Forwarded-For')
 
+    def _extract_post(self, *args, **kwargs):
+        """
+        Extracts post data (in json format) for the given platform.
+
+        Not all extractors will implement/require this method.
+        The method args and kwargs are up to the extractor to define as per what's needed for the given platform.
+        """
+        raise NotImplementedError
+
```

Comment:
no, we're not doing this

as I said before, extractor code is never public API. we are not standardizing this and setting the expectation that external apps directly calling extractor methods is a supported use case

revert this